src/db_connection/db_connect.ts

```diff
@@ -6,25 +6,22 @@ const connectToDatabase = async () => {
     const env = nconf.get('NODE_ENV') || 'development';
     const db = nconf.get(`db`);
     const dbEnvConfig = db[env]
-    console.log(env, "${env}");
-    console.log(`db:${env}`, "db:${env}");
-    console.log(db, "dbConfig");
-    console.log(dbEnvConfig, "dbConfig env");
 
     // Construct connection URL
     const MONGODB_URI: string = `${nconf.get(dbEnvConfig.mongo_connection_string)}://${nconf.get(dbEnvConfig.root_username)}:${nconf.get(dbEnvConfig.root_password)}@${nconf.get(dbEnvConfig.mongo_host_port)}/`
-    console.log(MONGODB_URI, "MongoDB");
 
     try {
-        await mongoose.connect(MONGODB_URI, dbEnvConfig.options);
+        let mongooseConnect  =  await mongoose.connect(MONGODB_URI, dbEnvConfig.options);
         console.log('Mongo connected');
         // Enable Mongoose debug mode if configured
         if (dbEnvConfig.debug) {
             mongoose.set('debug', true);
         }
+        return mongooseConnect;
```

Comment:
question: Why we need return mongooseConnect

Reply:
to return a promise and wait for it's execution